### ISSUE TITLE
Validation error Website must be valid

### ISSUE THREAD

**Issue by azhilt** [XeroAPI/Xero-NetStandard#434]
SDK Version: 3.26.0

The 'Website must be valid' validation error occurs when trying to save Xero.NetStandard.OAuth2.Model.Accounting.Contact using the AccountingApi.UpdateOrCreateContactsAsync method, if the contact has Web site value without http:// or https:// prefix in Xero.

The Contact object has a read-only WebSite property. Even when I set it to null or a value with http:// using reflection, the validation error occurs.

Error details:
{
  "ErrorNumber": 10,
  "Type": "ValidationException",
  "Message": "A validation exception occurred",
  "Elements": [
    {
      "ContactID": "XXX",
      "ContactStatus": "ACTIVE",
      "Name": "XXX",
      ...
      "UpdatedDateUTC": "\/Date(1661272953140+0000)\/",
      "ContactGroups": [],
      "IsSupplier": false,
      "IsCustomer": true,
      "Website": "www.xxx.net",
      "SalesTrackingCategories": [],
      "PurchasesTrackingCategories": [],
      "ContactPersons": [],
      "HasValidationErrors": true,
      "ValidationErrors": [
        {
          "Message": "Website must be valid."
        }
      ]
    }
  ]
}

**Comment by RettBehrens** [comment]
Hey @azhilt thanks for bringing this to our attention. I have confirmed the issue exists at the API level and is not specific to the SDK. It seems Xero UI allows a contact to be created with a website that the public API considers invalid. I have shared this this with the relevant team. Temporary workaround is to edit the contact details in Xero UI such that the website includes `http://` or `https://` as this will resolve the error allowing proper edits to the contact via the API again.
